arrpy/random.py

```
@@ -0,0 +1,97 @@
+            Random seed
+        """
+        self.seed(seed)
+    
+    def seed(self, seed=None):
+        """Set the random seed."""
+        _random.seed(seed)
+    
+    def uniform(self, low=0.0, high=1.0, size=None):
+        """
+        Generate uniform random numbers.
+        
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)